tests/cli/test_compose_live-iso.sh

```diff
@@ -51,7 +51,8 @@ rlJournalStart
 
     rlPhaseStartTest "Verify VM instance"
         # verify we can login into that instance *WITHOUT* a password
-        rlRun -t -c "ssh -oStrictHostKeyChecking=no -p 2222 root@localhost 'cat /etc/redhat-release'"
+        . ./tests/cli/lib/root_account.sh
+        check_root_account_live_iso root localhost "-p 2222 -i $SSH_KEY_DIR/id_rsa"
```

Comment:
I've tried this PR manually with the changes for enabling passwordless logins. You have to remove `i $SSH_KEY_DIR/id_rsa` from here. Very likely `-o PubkeyAuthentication=no` as well and we should be good to go.

Reply:
I updated the code, now when I see your comment again, I'm not sure what exactly you mean by 'changes for enabling passwordless logins'. Are those the changes done in sshd config that we agreed upon should be removed, or you added keys to the live iso when you tried the PR? At this stage the changes don't work for me, I can't think of any other way for the ssh login to work other than changing sshd config or adding the public key in the composed iso.